### PR TITLE
Changes to the QS

### PR DESCRIPTION
**Pull request by Andrew-Pohl**
• Check if node_key is non default, exit if default.
• Add a new function "displayErrorAndExit" echos and error in red and exits.
• Fixed bug in node creation

### CODE REVIEW

**Comment by Andrew-Pohl** [comment]
fix for #76 and other little bugs

File: scripts/quickstart.sh [hunk 1]
```diff
@@ -467,7 +473,7 @@ function run {
       if [ $PLATFORM == "LINUX" ]; then
         cpuCores=$(nproc --all)
       fi
-      NUM_RPC_THREADS=cpuCores
+      NUM_RPC_THREADS=$cpuCores
       NUM_HTTP_THREADS=$(( 4*cpuCores ))
```
Comment (leonprou) [inline scripts/quickstart.sh:477]: @Andrew-Pohl need o add this here too :)

**Review by leonprou — CHANGES_REQUESTED** [review]

File: scripts/quickstart.sh (hunk 1 above)
Comment (Andrew-Pohl) [reply scripts/quickstart.sh:477]: The arthmetic expression should mean you don't have to there. (They are both technically right but not needed)

Comment (leonprou) [reply scripts/quickstart.sh:477]: great, was making sure you didn't miss it

**Review by leonprou — APPROVED** [review]

**Pull request merged**